src/main/java/model/data/process/functions/Minmax.java

```diff
@@ -0,0 +1,51 @@
+package model.data.process.functions;
+
+import model.data.DataTable;
+import model.data.describer.DataDescriber;
+import model.data.value.DataValue;
+import model.data.value.FloatValue;
+import model.data.value.NumberValue;
+
+/**
+ * This class will provide a framework for functions resulting a datavalue.
```

Comment:
Hij is eigenlijk alleen voor min en max. Wat je wel kan doen is een framework maken voor een reduce functie https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce